libraries/src/Component/ComponentHelper.php

```diff
@@ -172,7 +172,7 @@ public static function filterText($text)
 			}
 			else
 			{
-				// Blacklist or whitelist.
+				// DisallowList or AllowList.
```

Comment:
```suggestion
				// Disallow List or Allow List.
```
Formalise the wording to show as separate words.